### ISSUE TITLE
Finalize heatmap and download functionality

### ISSUE THREAD

**Issue by jessicaguo** [cct-datascience/organization#921]
- [x] Incorporate doi info into database
- [x] Develop export buttons for matrix and long
- [ ] Try 'fastmatch' package in place of %in%
- [x] adjust UI to allow for blank as input to dropdowns
- [ ] set NA values to gray, and re-label scale bar with >= and <= 2
- [ ] Send email to whole team with link to prototype app, appreciate feedback, and inquire about proposals/deadlines

**Comment by dlebauer** [comment]
* [ ] For heatmap x-axis, the experiment ID should be uniquely identifiable - perhaps by appending the treatment / condition?
* [ ] Update README w/ content to date, future work, sources of data, contributors, other information

**Comment by jessicaguo** [comment]
Plotly has mouseover - vs Shiny tool tip?

**Comment by jessicaguo** [comment]
Stuck on the >= and <= symbols in the color bar and the persistent error messages that show up, e.g.
![image.png](https://images.zenhubusercontent.com/5f513e94d41e1d43610542ee/d6cc0232-5e4e-437b-a039-1817973d48f4)
They don't seem to affect the app performance, but they are very disconcerting. Maybe @Aariq can help me debug next week?

**Comment by Aariq** [comment]
I'm think I know where to look for a solution for this, so I can give it a shot.

**Comment by Aariq** [comment]
The answer is here: https://youtu.be/87rpiuRyhaQ?t=821

"You have to check yourself before you `req()` yourself"

Inside of a `render*()` function, `req()` will prevent it from doing anything unless whatever is in `req()` exists, I think.

**Comment by jessicaguo** [comment]
Thanks Eric! Great solve. 
I'm tinkering with color on the heatmap. Bryan wanted the NA spaces to be gray. Option 1 I include the NA's and set na.value in the `scale_fill_distiller()`; however, the NAs then register when clicking, which I do not want. So I went with Option 2, setting the background fill color to gray, but since I'm using transparency as a way of distinguishing between significant and non-significant log2 ratios, the color isn't what I envisioned. Currently setting alpha = 0.3 for log2 values below the threshold.
